recipes/pygnome/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: f4263ea7e373414ea66ca0f630b705b07e0131258b2fafad03d369b0f5ef0ee6
+
+build:
+  skip: true  # [linux]
```

Comment:
why is linux skipped?

Reply:
It is skipped because we didn't verify it on linux.  Don't have an environment for it.  Although our Gitlab runners are on CentOS.  I will enable it just to see if it works.